test/clientconn_state_transition_test.go

```diff
@@ -616,3 +618,74 @@ func (s) TestConnectivityStateSubscriber(t *testing.T) {
 		}
 	}
 }
+
+// TestChannelStateWaitingForFirstResolverUpdate verifies the initial
+// state of the channel when a manual name resolver doesn't provide any updates.
+func (s) TestChannelStateWaitingForFirstResolverUpdate(t *testing.T) {
+	t.Skip("The channel remains in IDLE until the LB policy updates the state to CONNECTING. This is a bug and the channel should transition to CONNECTING as soon as Connect() is called.")
```

Comment:
Doug mentioned that we should  reference the Github issue in the skip message. Can you please add the line "See issue #7686." as the end?

```go
t.Skip("... soon as Connect() is called. See issue #7686.")
```
Same change for the other test case also.